### PR TITLE
Add checklist for new lab members

### PR DESCRIPTION
**Pull request by miltondp**
This PR adds a checklist to the `README.md` file so new lab members have a better idea of the common tasks necessary to fully join the laboratory.

The reviewer of this PR should be a current lab member and follow the instructions in the checklist, and provide feedback on whether they are clear and work as expected.

### CODE REVIEW

File: README.md
Comment (msubirana) [inline README.md:1]: perhaps mention how to request access to the Alpine cluster?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+# Checklist for new lab members
+* **Read the [`onboarding.md`](onboarding.md) document**.
+It has important information about the lab's mission, expectations, lab meetings, indiviudal meetings, source code, reproducibility, etc.
+Ask if you have questions.
+* Request access to the [lab's GitHub organization](https://github.com/pivlab).
+For this, Milton will need your GitHub user.
+* Request access to the [lab's Slack workspace](https://pivlab.slack.com/).
+It is usually a good idea to link this account to your CU email address.
+* 
+
```
Comment (msubirana) [inline README.md]: perhaps mention how to request access to the Alpine cluster or add the links with the information?
add how to set up CU VPN, anydesk VPN and rustdesk tunnel?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+For this, Milton will need your GitHub user.
+* **Request access to the [lab's Slack workspace](https://pivlab.slack.com/)**.
+We need your email address to send you an invitation to join.
+* **Add yourself to the [lab's team webpage](https://pivlab.org/team/)**.
+For this, you need to fork the [lab's webpage GitHub repo](https://github.com/pivlab/pivlab-website), make the necessary changes, and then submit a Pull Request (PR).
+Follow [the documentation on how to add team members](https://greene-lab.gitbook.io/lab-website-template-docs/basics/team-members) and/or take a look at [this PR](https://github.com/pivlab/pivlab-website/pull/4).
+* (optional) **Add your recent publications to the lab's website**.
+If you want visitors to know more about your research, you can add your recent publications to the lab's website.
+This usually involves modifying the `_data/sources.yaml` file.
+See [the documentation on citations](https://greene-lab.gitbook.io/lab-website-template-docs/basics/citations) for more information.
```
Comment (msubirana) [inline README.md:23]: perhaps mention how to request access to the Alpine cluster or add the links with the information?
add how to set up CU VPN, anydesk VPN and rustdesk tunnel?

File: README.md [hunk 3]
```diff
@@ -18,6 +33,9 @@ Consider the following goals of the onboarding document:
 + Living: The document is expected to change.
 + Local Resources: (should this be part of a separate document?)
 
+## Useful links
+* [Pividori Lab logos](https://github.com/pivlab/pivlab-website/tree/main/images/logo).
+
```
Comment (msubirana) [inline README.md:38]: we'll need a picture of the group when Haoyu joins the lab!

File: README.md (hunk 1 above)
Comment (miltondp) [reply README.md]: Great point. I added a document at [docs/cu_dbmi.md](docs/cu_dbmi.md)

File: README.md (hunk 3 above)
Comment (miltondp) [reply README.md:39]: Absolutely!

**Pull request merged**